<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>立方体中点迭代</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labellogo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>640</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/image/horizontal.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusmsg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>68</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="openglwidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>141</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>0</y>
       <width>121</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>58</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxGeneration">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>20</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>显示模式</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/sign.png</normaloff>:/image/sign.png</iconset>
     </property>
     <addaction name="actionSmoothShaded"/>
     <addaction name="actionFlatShaded"/>
     <addaction name="actionHiddenLine"/>
     <addaction name="actionWireframe"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionimport"/>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionimport">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/sign.png</normaloff>:/image/sign.png</iconset>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/sign.png</normaloff>:/image/sign.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionSmoothShaded">
   <property name="text">
    <string>平滑</string>
   </property>
  </action>
  <action name="actionFlatShaded">
   <property name="text">
    <string>平滑阴影</string>
   </property>
  </action>
  <action name="actionHiddenLine">
   <property name="text">
    <string>隐藏线</string>
   </property>
  </action>
  <action name="actionWireframe">
   <property name="text">
    <string>线框</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>openglwidget</class>
   <extends>QWidget</extends>
   <header>openglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
